### ISSUE TITLE
Section 12.2 uses Sys.delay() instead of Sys.sleep()

### ISSUE THREAD

**Issue by nickv9** [hadley/adv-r#1024]
[Section 12.2(Behavioural FOs)](https://adv-r.hadley.nz/function-operators.html#behavioural-fos) in the online version contains a code snipped which calls Sys.delay(1) - I *think* the desired call is Sys.sleep(1).

    i <- 1
    for(url in urls) {
      i <- i + 1
      if (i %% 10 == 0) cat(".")
      Sys.delay(1) <--shouldnt this be Sys.sleep(1)?
      download_file(url)
    }

**Comment by jdblischak** [comment]
@nickv9 I agree this looks like a typo. Do you want to send a Pull Request to fix it? You can edit the Rmd file and submit a Pull Request directly on GitHub by clicking on the icon that looks like a pencil in a square.

![screenshot from 2018-06-26 09-51-28](https://user-images.githubusercontent.com/1608317/41916738-ae5132d0-7926-11e8-8a48-e29acf68ed5a.png)

**Comment by nickv9** [comment]
@jdblischak I've never done a pull request but I will give it a shot next week - good practice!

**Comment by nickv9** [comment]
Just FYI: I didn't get it done this week but I still plan to. I spent a few hours this week working through http://happygitwithr.com/ to learn more about git.

**Comment by jdblischak** [comment]
@nickv9 Congrats on having your first PR merged! May it be the first of many :-)

<img width="626" alt="screen shot 2018-07-11 at 6 34 09 pm" src="https://user-images.githubusercontent.com/1608317/42602795-1e967370-8539-11e8-82fa-667a144bacf5.png">
